### ISSUE TITLE
write docs for ways a dev can get postgresql running

### ISSUE THREAD

**Issue by boazsender** [bocoup/image-annotator#126]

**Comment by mzgoddard** [comment]
```
docker run -n ia-pq -d -p 5432:5432 postgres
docker exec -it iq-pq "su postgres -c 'createdb image-annotations'"
PGUSER=postgres npm migrate
PGUSER=postgres npm start
```

Here is how I'm deving with postgres from a docker image.

**Comment by boazsender** [comment]
Resolved by #180
